install/firmware.csv

```diff
@@ -1131,7 +1131,8 @@ Trimble Navigation Ltd.,TRIMBLE NETRS,4350230198,1.1-3,2006-03-10T12:38:43Z,2006
 Trimble Navigation Ltd.,TRIMBLE NETRS,4350230198,1.1-5,2006-04-03T14:44:32Z,9999-01-01T00:00:00Z,
 Trimble Navigation Ltd.,TRIMBLE NETRS,4351230275,1.0-2,2004-02-03T00:00:00Z,2004-05-05T03:09:00Z,
 Trimble Navigation Ltd.,TRIMBLE NETRS,4351230275,1.0-3,2004-05-05T03:09:15Z,2006-05-21T21:11:30Z,
-Trimble Navigation Ltd.,TRIMBLE NETRS,4351230275,1.1-5,2006-05-21T21:11:49Z,9999-01-01T00:00:00Z,
+Trimble Navigation Ltd.,TRIMBLE NETRS,4351230275,1.1-5,2006-05-21T21:11:49Z,2019-03-04T04:36:00Z,
+Trimble Navigation Ltd.,TRIMBLE NETRS,4351230275,1.3-2,2019-03-04T04:36:01Z,9999-01-01T00:00:00Z,
```

Comment:
Should the start date of this firmware be `2019-04-04T03:36:01Z`? It looks like the month and day are swapped in this PR